samples/Bluetooth/BluetoothLESample3/Services/BatteryService.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using nanoFramework.Device.Bluetooth;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+
+namespace nanoFramework.Device.Bluetooth.Services
+{
+    class BatteryService
```

Comment:
```suggestion
    private class BatteryService
```

Reply:
That doesn't work having it private

Reply:
Fine, just add the proper visibility. Note that I've added several other that where missing. Please check.